tools/check_volume_levels.m

```diff
@@ -282,3 +279,26 @@ function plot_levels(meas, tc, lm)
 		y(:,j) = filter(b, a, x(:,j));
 	end
 end
+
+function x = multitone( fs, f, amp, tlength )
+	n = round(fs*tlength);
+	t = (0:n-1)/fs;
+	nf = length(f);
+	if nf > 1
+		ph = rand(nf, 1)*2*pi;
+	else
+		ph = 0;
+	end
+
+	x = zeros(n, 1);
+	x(:,1) = amp(1)*sin(2*pi*f(1)*t);
+	for i=2:length(f)
+		x(:,1) = x(:,1) + amp(i)*sin(2*pi*f(i)*t+ph(i))';
+	end
+end
+
+function dbfs = level_dbfs(x)
```

Comment:
Can you add a one-line comment with a pointer to the copies in sof? "A little copying is better than a little dependency." but it's even better when people are aware :-)

Reply:
Yep, I added the the notes.